src/js/index.js

```diff
@@ -87,9 +89,12 @@ function cascade(force = false) {
 window.addEventListener('load', () => {
 	selectr = new Selectr('#tags', {
 		multiple: true,
+		searchable: false,
 		placeholder: 'Choose tags...',
 		data: window.swagTags.map(tag => ({value: tag, text: tag}))
 	});
+	tagsPlaceholderSelect.style.display = 'none';
```

Comment:
I think we should hide the tags search and enable it when Selectr loads - thoughts?

Reply:
Inlining the tag selector is done in #159